audio/channel_receive.cc

```diff
@@ -347,6 +350,11 @@ AudioMixer::Source::AudioFrameInfo ChannelReceive::GetAudioFrameWithInfo(
     return AudioMixer::Source::AudioFrameInfo::kError;
   }
 
+  /*if (muted) {
+      RTC_LOG(LS_INFO) << "MUTEEEEEEED!!!!!!!!!";
+  } else {
+      RTC_LOG(LS_INFO) << "NIJE MUTED!!!!!!!!!";
+  }*/
```

Comment:
Should this be removed?

Reply:
Yes, I will remove it.

Reply:
Actually should be already removed here: https://github.com/coretech/webrtc/pull/1/commits/f6030c20b950b3f564a5bf1b86f4fef4e04d39bc